### DISCUSSION TITLE
Autocomplete project based on filter

### DISCUSSION THREAD

**Discussion by tescaso** [ransome1/sleek#357] [Ideas]
I am not sure if this has already been considered:
 Is there a way to configure it so, when you have only one project filtered, the creation of a new task has this project appended automatically?

**Comment by ransome1** [comment]
@ryuitxi I think that might be covered in this feature request: https://github.com/ransome1/sleek/issues/281

**Reply by tescaso** [reply]
Yes, it is. Thank you.
